pkgs/top-level/python-packages.nix

```diff
@@ -11890,6 +11890,20 @@ in modules // {
     };
   });
 
+  libgpuarray-cuda = callPackage ../development/python-modules/libgpuarray/cuda/default.nix rec {
+    numpy = self.numpy_1_10;
+    scipy = self.buildScipyPackage rec {
+      version = "0.17.1";
+      src = pkgs.fetchurl {
```

Comment:
You are correct. What you had originally seems prettier then.

Anyway, please don't select specific versions of a Python package here, because as a user you will eventually end up with multiple versions in a closure when adding other packages that depend on `numpy`/`scipy` to your environment.